samples/image-metadata/src/components/FocalPointDialog/FocalPointDialog.js

```diff
@@ -55,6 +54,11 @@ export class FocalPointDialog extends Component {
   };
 
   onImageClick = e => {
+    const imageWasClicked = this.imgRef.current === e.target;
```

Comment:
Is `this.imgReg` always going to exist? Should we check it's there first before trying to access `current`?